src/adapter/src/session/vars.rs

```diff
@@ -1559,6 +1578,18 @@ impl SystemVars {
     pub fn mock_audit_event_timestamp(&self) -> Option<mz_repr::Timestamp> {
         *self.mock_audit_event_timestamp.value()
     }
+
+    /// Returns the `enable_with_mutually_recursive` configuration parameter.
+    pub fn enable_with_mutually_recursive(&self) -> bool {
+        *self.enable_with_mutually_recursive.value()
+    }
+
+    /// Sets the `enable_with_mutually_recursive` configuration parameter.
+    pub fn set_enable_with_mutually_recursive(&mut self, value: bool) -> bool {
+        self.enable_with_mutually_recursive
+            .set(VarInput::Flat(if value { "true" } else { "false" }))
```

Comment:
Does `VarInput::Flat(value.format())` work instead of `VarInput::Flat(if value { "true" } else { "false" })` since `Value` is implemented for `bool` below?

Reply:
Ah, nice catch!